content/en/tracing/version_tracking/_index.md

```diff
@@ -14,6 +14,75 @@ further_reading:
+
+- Total Errors by Version
+- Errors per second by Version
+- % Error rate by Version
+
+All of these can be exported to dashboards and monitors.
+
+## Version Summary Table
+
+Services configured with version tags will have a `version` section on the Service page, immediately below the main service health graphs.  This table will show all versions of the service that were active during the selected time interval, with currently active services at the top.
+
+By default you will see:
+
+- The version name(s) deployed for this service over the timeframe.
+- The times traces corresponding to this version was first and last seen.
+- An indication of how many types of errors appear in each version that did not appear in the immediately previous version.  Note that this is a not a guarantee the version introduced these errors, but an indication that this error was not seen for the previous version.  Based on how you are performing deployments, it can be a great way to begin investigating errors.
+- Requests per second.
+- Error rate as a percentage of total requests.
+
+Additionally, fields can be added or removed from this overview table and your selections will be saved.  The additional fields are:
+
+- Endpoints that appear in a version that did not appear in the previous version.
+- Time active, showing the length of time from the first trace to the last trace sent to Datadog for that version.
+- Total number of Requests.
+- Total number of Errors.
+- Latency measured by p50, p75, p90, p95, p99, or the true max.
+
+{{< img src="tracing/version_tracking/VersionComparison.png" alt="Versions on the Service Page"  style="width:100%;">}}
+
+**Note:** The `version` section of of the service page will only appear if there is more than one version reporting in the selected time interval at the top of the page.
+
+## Version Comparison
+
+Clicking on any version row in the Version Summary table will open a detailed version comparison page, allowing you to compare two specific version of the same service.  By default, the comparison will compare the version clicked to the immediately previous version, but this can be changed to compare any two versions within the past 30 days.
+
+There are three main components of version comparison:
+
+- [Comparison Graphs](#comparison-graphs) for a visualization of requests and errors to services, useful for watching various types of [deployments](#version-deployment-strategies).
+- [Error Comparison](#error-comparison) to see errors that may have been introduced or solved by a deployment.
+- [Endpoint Comparison](#endpoint-comparison) to see how endpoint latency and error rates are performing in different versions of the service.
+
+### Comparison Graphs
+
+Similar graphs to the main service page will appear, however all other versions than the two being specifically compared will be combined and labelled as 'Other versions', allowing a quick overview of a deployment or difference in error rate to be quickly noticed.
```

Comment:
How about: "Requests and Error graphs are available in this section, much like the graphs on the Service Page. These graphs highlight the selected versions for comparison but leave all other versions in gray for additional context. These allow for a quick overview of a deployment rollout or the identification of a spike in error rates."